doi.jq

```diff
@@ -54,10 +53,8 @@ def repos_anti_subset:
 		"mediawiki",
 		"monica",
 		"nextcloud",
-		"python", # direct children: hylang plone satosa unit
```

Comment:
No SBOM for python?

Reply:
Python was accidentally added to the SBOM list is seems but it wouldn't get them without being on the new build

Reply:
AAaaah, good point. It never occured to me to do it that way